### PR TITLE
Minor Justice Fixes

### PR DESCRIPTION
**Pull request by Lyciata**
## About the PR
I really just made it so now CJ nows need Clerk time again, gave CJ a flash since they were the only command with no flash and I put the Clerk's outfits inside the uniform printer

## Why / Balance
Just some minor details that I felt like could be changed

## Technical details
Haven't tried nor can I try so let's pray it works!

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
Can't test it so I can't say if it will actually works

**Changelog**
:cl:
- add: Added Clerk suit and skirt into the Uniform Printer
- tweak: Chief Justice now need Clerk time again
- tweak: Chief Justice will now spawn with a flash in their backpack

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Roles/Jobs/Justice/chief_justice.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@
   description: job-description-chief-justice
   playTimeTracker: JobChiefJustice
   requirements:
-#    - !type:RoleTimeRequirement # DeltaV - Comment out clerk time till more people have clerk time. Maybe a week or so. 
-#      role: JobClerk
-#      time: 36000 # DeltaV - 10 hours
+    - !type:RoleTimeRequirement 
+      role: JobClerk
+      time: 36000 # DeltaV - 10 hours
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Roles/Jobs/Justice/chief_justice.yml]: dont need deltav comment since its in deltav file

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by deltanedas — APPROVED** [review]

**Comment by deltanedas** [comment]
all you need to do is add recipes in deltav folder and its good

**Comment by TadJohnson00** [comment]
@Lyciata is this still being worked on?

**Comment by deltanedas** [comment]
stale

**Pull request closed without merging**